airbyte-cdk/python/airbyte_cdk/utils/schema_inferrer.py

```diff
@@ -53,23 +73,15 @@ class SchemaInferrer:
 
     stream_to_builder: Dict[str, SchemaBuilder]
 
-    def __init__(self) -> None:
+    def __init__(self, pk: List[List[str]] = None, cursor_field: List[List[str]] = None) -> None:
         self.stream_to_builder = defaultdict(NoRequiredSchemaBuilder)
+        self._pk = pk
+        self._cursor_field = cursor_field
 
     def accumulate(self, record: AirbyteRecordMessage) -> None:
         """Uses the input record to add to the inferred schemas maintained by this object"""
         self.stream_to_builder[record.stream].add_object(record.data)
 
-    def get_inferred_schemas(self) -> Dict[str, InferredSchema]:
```

Comment:
This was used only for test purposes so I decided it would be better to just remove it